src/generators/default/number/number.test.ts

```diff
@@ -82,17 +82,17 @@ describe('create numbers', () => {
 
 	describe('multipleOf', () => {
 		test('produces a valid multipleOf number', () => {
-			expect(core).toProduce(z.number().multipleOf(33));
+			expect(core).toReasonablySatisfy(z.number().multipleOf(33));
 		});
 
 		test("creates a number that's a multiple with min and max", () => {
 			const schema = z.number().multipleOf(3).min(16).max(20);
-			expect(core).toProduce(schema, 18);
+			expect(core).toReasonablySatisfy(schema, 18);
```

Comment:
This needs to be updated.
18 was the expected result, but is now passed as the number of iterations.